src/ontology/uberon-edit.obo

```diff
@@ -224246,6 +224246,119 @@ intersection_of: part_of UBERON:0001987 ! placenta
+synonym: "ischio-rectal perianal space" EXACT [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+is_a: UBERON:8480009 ! perianal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T10:46:32Z xsd:dateTime
+
+[Term]
+id: UBERON:8480015
+name: superior pelvi-rectal perianal space
+def: "A perianal space located above the anal canal and is delimited by the superior pelvic aponeurosis below, by the peritoneum above and by the rectum internally." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+is_a: UBERON:8480009 ! perianal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T10:59:37Z xsd:dateTime
+
+[Term]
+id: UBERON:8480016
+name: perirenal space
+def: "A retroperitoneal space that lies between the anterior and posterior renal fascia; it is a paired inverted cone shaped of tissue containing the kidney, the proximal portion of the collecting system, renal and perirenal vasculature, perirenal lymphatics, adrenal glands and variable amount of fat." [PMID:24065628]
+synonym: "perinephric space" EXACT [PMID:28050791]
+is_a: UBERON:0003693 ! retroperitoneal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T11:02:35Z xsd:dateTime
+
+[Term]
+id: UBERON:8480017
+name: anterior pararenal space
+def: "A space that is bounded anteriorly by the posterior parietal peritoneum, laterally by the lateroconal fascia and posteriorly by the anterior renal fascia; it contains the pancreatic gland, retroperitoneal segments of the duodenum, ascending and descending colon and the mesenteric root, as well as adipose tissue." [PMID:24065628]
+is_a: UBERON:0003693 ! retroperitoneal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T11:19:27Z xsd:dateTime
+
+[Term]
+id: UBERON:8480018
+name: posterior pararenal space
+def: "A space that is bound anteriorly by the posterior renal fascia and posteriorly by the transversalis fascia, and it only contains adipose tissue, lymphatics and blood vessels." [PMID:24065628]
+is_a: UBERON:0003693 ! retroperitoneal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T11:20:55Z xsd:dateTime
+
+[Term]
+id: UBERON:8480019
+name: perirectal space
+def: "An extraperitoneal space between the rectal fascia and the rectal wall. Limited by the rectovaginal (in females) or rectovesical (in males) septum, and the visceral endopelvic fascia." [PMID:24065628]
```

Comment:
Consider this definition, which may be more clear:
An extraperitoneal space bounded laterally by the perirectal fascia, posteriorly by the posterior pelvic fascia, and anteriorly by the rectovesical fascia in the male and the rectovaginal fascia in the female. [PMID:26006061](https://pubmed.ncbi.nlm.nih.gov/26006061)

Reply:
@bvarner-ebi Thanks, I changed it